ui/app/src/components/PathNavigator/PathNavigator.tsx

```diff
@@ -168,6 +186,12 @@ export default function PathNavigator(props: PathNavigatorProps) {
     }
   }, [dispatch, excludes, id, limit, locale, path, site, state, uiConfig.currentSite]);
 
+  useMount(() => {
+    if (state) {
+      dispatch(pathNavigatorBackgroundRefresh({ id }));
+    }
+  });
+
```

Comment:
Could we have this as part of the same effect that initialises?

Reply:
no, because the other effect is listening the state, and it changes very often